pkg/daemon/ceph/client/upgrade_test.go

```diff
@@ -69,12 +82,90 @@ func TestEnableNautilusOSD(t *testing.T) {
 	executor.MockExecuteCommandWithOutput = func(debug bool, name string, command string, args ...string) (string, error) {
 		assert.Equal(t, "osd", args[0])
 		assert.Equal(t, "require-osd-release", args[1])
-		assert.Equal(t, "nautilus", args[2])
-		assert.Equal(t, 3, len(args))
 		return "", nil
 	}
 	context := &clusterd.Context{Executor: executor}
 
 	err := EnableNautilusOSD(context)
 	assert.Nil(t, err)
 }
+
+func TestOkToStopDaemon(t *testing.T) {
+	executor := &exectest.MockExecutor{}
+	executor.MockExecuteCommandWithOutput = func(debug bool, name string, command string, args ...string) (string, error) {
+		assert.Equal(t, "mon", args[0])
+		assert.Equal(t, "ok-to-stop", args[1])
+		assert.Equal(t, "a", args[2])
+		return "", nil
+	}
+	context := &clusterd.Context{Executor: executor}
+
+	deployment := "rook-ceph-mon-a"
+	err := okToStopDaemon(context, deployment, "rook-ceph")
+	assert.Nil(t, err)
+}
+
+func TestFindDaemonName(t *testing.T) {
+	n, err := findDaemonName("rook-ceph-mon-a")
+	assert.Nil(t, err)
```

Comment:
`assert.NoError` is more clear than `assert.Nil`

Reply:
`s/assert.Nil(t, err)/assert.NoError(t, err)/g` & `s/assert.NotNil(t, err)/assert.Error(t, err)/g`